xbmc/guilib/guiinfo/MusicGUIInfo.cpp

```diff
@@ -75,7 +75,8 @@ bool CMusicGUIInfo::InitCurrentItem(CFileItem *item)
 
 bool CMusicGUIInfo::GetLabel(std::string& value, const CFileItem *item, int contextWindow, const CGUIInfo &info, std::string *fallback) const
 {
-  if (info.GetData1() && info.m_info >= MUSICPLAYER_TITLE && info.m_info <= MUSICPLAYER_ALBUM_ARTIST)
+  // For musicplayer info labels check playlist
+  if (info.GetData1() && info.m_info >= MUSICPLAYER_TITLE && info.m_info <= MUSICPLAYER_DBID)
```

Comment:
`if (info.GetData1() && info.m_info >= MUSICPLAYER_OFFSET_POSITION_FIRST && info.m_info <= MUSICPLAYER_OFFSET_POSITION_LAST)` ?

Once those deines are in places, this code piece never needs to be adjusted again (if labels are added/removed).